src/dbcp/transform/acp_projects.py

```diff
@@ -479,4 +486,64 @@ def _str_strip(ser):
     }
     out.rename(columns=rename_dict, inplace=True)
     out.drop(columns=cols_to_drop, inplace=True)
-    return {"acp_projects": out}
+    return out
+
+
+def _clean_col_names_and_create_id(df) -> pd.DataFrame:
+    """Create surrogate key based on raw columns, then rename and assign ID.
```

Comment:
I think we should make it clear whether we are creating an ID or a state hash in order to track state changes.

Reply:
This ID creation is actually from the previous ACP table creation (not change log) code. It creates a `proj_id` surrogate key based on some of the columns, because we lack a primary key to identify each project in the raw data. 

When creating the change log, I still wanted to create + assign the `proj_id` to each row. Then, I group by this `proj_id` and hash each row (which hashes more columns than are used int he creation of the `proj_id`). So in my mind, it's necessary to have the `proj_id` (which connects projects to other ACP tables and uniquely identifies a project), and a state hash (which is a hash of all columns, not just those used to create the surrogate key `proj_id`).